content/Hardware Support/MKR Family/About-the-IoT-Bundle-projects.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "About the IoT Bundle projects"
+id: 4407181791762
+---
+
+Both **Arduino IoT Bundle** and **Arduino _MKR_ IoT Bundle** include components that can be used for five projects.
+
+---
+
+## Arduino IoT Bundle
+
+Tutorials for the **Arduino IoT Bundle** projects are available in Arduino Docs:
```

Comment:
should we add the Arduino Docs link as a Hyperlink?

Reply:
Since the article is about the projects, I felt it might be better that the users use links in the project list. If we provide a link to Docs they may follow it and be confused how to open the actual projects, whereas they can always open a project and continue exploring Docs from there. Same for Project Hub with [this comment](https://github.com/arduino/help-center-content/pull/159/files#r1060506040)